solidity/package.json

```diff
@@ -56,15 +53,14 @@
     "bn.js": "^4.11.8",
     "chai": "^4.2.0",
     "eslint": "^5.16.0",
-    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.2.0",
-    "eslint-config-prettier": "^6.10.0",
+    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.3.0",
```

Comment:
Something's odd with the version of this dependency in the lock file. It's still referencing the commit of the `0.2.0` tag, not `0.3.0`. The rules that were used to lint the other files seem to be pulled from the previous config version.

Reply:
I think I see what you're talking about. In `devDependencies` we have 
```
"eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.3.0",
```
but in `packages.node_modules/eslint-config-keep` we have:
```
"resolved": "git+ssh://git@github.com/keep-network/eslint-config-keep.git#c2502e8e677e58e0e1c91ea0312849fbda31f1d8",
```
which refers to `0.2.0` <_<

same later on in the file:
```
    "eslint-config-keep": {
      "version": "git+ssh://git@github.com/keep-network/eslint-config-keep.git#c2502e8e677e58e0e1c91ea0312849fbda31f1d8",
```

Reply:
Maybe this is related:
![image](https://user-images.githubusercontent.com/1045160/118326292-c9ff0f80-b4d2-11eb-9e22-94987bd463e3.png)

Reply:
Why does that have two tags? When I click 0.3.0, it takes me to https://github.com/keep-network/eslint-config-keep/releases/tag/0.3.0 which resolves to a different commit:

![image](https://user-images.githubusercontent.com/1045160/118326429-f7e45400-b4d2-11eb-83da-8029880d5985.png)